Url password can have quoted special characters.

javier-sanz opened the pull request:
As described on this Redash [forum support issue](https://discuss.redash.io/t/redis-quoted-password/1435) password can have special characters and those ones can be `quoted` (encoded). They must be `unquoted ` before being sent otherwise the validation process will fail.

arikfr commented on redash/__init__.py:
```diff
@@ -53,8 +54,11 @@ def create_redis_connection():
+        redisPassword = ""
+        if redis_url.password is not None:
+            redisPassword = urllib.unquote(redis_url.password)
+        r = redis.StrictRedis(host=redis_url.hostname, port=redis_url.port, db=redis_db, password=redisPassword)
```
We use `snake_case` and not `camelCase`. And this can be simplified to:

```python
redis_password = redis_url.password and urllib.unquote(redis_url.password)
```

If the password is `None`, it will just assign `None`.

javier-sanz commented:
@arikfr I fixed it following your comments. I added another change on the helper.py that I think it has the same issue. Let me know if you thing that change is not needed.

arikfr commented:
Thanks !

javier-sanz commented:
NP 😃
